### PR TITLE
LIT5895 98 00

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New items, corrections.

### CODE REVIEW

File: 1001-2000/LIT1651Homily.xml [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
 type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="work" xml:lang="en" xml:id="LIT1651Homily">
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title xml:lang="gez" xml:id="t1">ድርሳን፡ ዘ፬እንስሳ፡ መንፈሳውያን፡ ወብርሃናውያን፡</title>
-            <title xml:lang="gez" corresp="#t1" type="normalized">Dǝrsān za-4 ʾǝnsǝsā manfasāwǝyān wa-bǝrhānāwǝyān</title>
-            <title xml:lang="en" xml:id="t2">Homily on the Four Living Creatures</title>
+           <title xml:lang="en" corresp="#t1">Homily on the Four Living Creatures by John Chrysostom</title>
```
Comment (thea-m) [inline 1001-2000/LIT1651Homily.xml:7]: Is the first line the title you want to have in the article?

File: 1001-2000/LIT1651Homily.xml [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
+           <title xml:lang="gez" xml:id="t2">ድርሳን፡ ዘ፬እንስሳ፡</title>
+           <title xml:lang="gez" corresp="#t2" type="normalized">Dǝrsān za-4 ʾǝnsǝsā</title>
+           <title xml:lang="en" corresp="#t2">Homily on the Four Living Creatures</title>
+
+            <title xml:lang="gez" xml:id="t3">ድርሳን፡ ዘ፬እንስሳ፡ መንፈሳውያን፡ ወብርሃናውያን፡</title>
+            <title xml:lang="gez" corresp="#t3" type="normalized">Dǝrsān za-4 ʾǝnsǝsā manfasāwǝyān wa-bǝrhānāwǝyān</title>
+            <title xml:lang="en" corresp="#t3">Homily on the Four Living Creatures</title>
```
Comment (thea-m) [inline 1001-2000/LIT1651Homily.xml:14]: manfasāwǝyān wa-bǝrhānāwǝyān should be translated...

File: 3001-4000/LIT3058RepCh338.xml [hunk 3]
```diff
@@ -1,16 +1,12 @@
...
       <fileDesc>
          <titleStmt>
-            <title xml:id="t1">Hymn to Our Lady Mary (ʾƎsaggǝd laki)</title>
-            
-            <title xml:id="t2" xml:lang="gez" type="main">እሰግድ፡ ለኪ፡ ወእዌድሰኪ፡
-            </title>
-            <title corresp="#t2" xml:lang="gez" type="normalized">ʾƎsaggǝd laki waʾǝweddǝsaki</title>
-            <title corresp="#t2" xml:lang="en">I worship
-               Thee…</title>
+            <title xml:lang="en" xml:id="t1" type="main">I worship Thee… </title>
```
Comment (thea-m) [inline 3001-4000/LIT3058RepCh338.xml:7]: If you want "I worship thee..." to be the printed title, I would suggest:
```
<title xml:lang="gez" xml:id="t1">እሰግድ፡ ለኪ፡ እሰግድ፡ ለኪ፡ እሰግድ፡ ለኪ፡ ወእዌድሰኪ፡ </title>
<title  xml:lang="gez" corresp="#t1" type="normalized">ʾƎsaggǝd laki ʾǝsaggǝd laki ʾǝsaggǝd laki wa-ʾǝweddǝsaki</title>
<title xml:lang="en" corresp="#t1">I worship you, I worship you, I worship you, and I praise you… </title>
<title xml:lang="en" corresp="#t1" type="main">I worship Thee… </title>
```

File: 4001-5000/LIT4958Priests.xml [hunk 4]
```diff
@@ -1,12 +1,13 @@
...
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng"
 type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT4958Priests" xml:lang="en" type="work">
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title xml:lang="gez" xml:id="t1">ድርሳን፡ በእንተ፡ ክብሮሙ፡ ለ፳፬፡ ካህናተ፡ ሰማይ፡</title>
-                <title xml:lang="gez" corresp="#t1" type="normalized">Dǝrsān baʾǝnta kǝbromu la-24 kāhǝnāta samāy</title>
-                <title xml:lang="en" corresp="#t1">Homily on the glory of the 24 Heavenly Priests</title>
+              <title xml:lang="en" corresp="#t1">Homily on the glory of the 24 Heavenly Priests by Cyril, Bishop of Jerusalem</title>
```
Comment (thea-m) [inline 4001-5000/LIT4958Priests.xml:7]: If "Homily on the glory of the 24 Heavenly Priests by Cyril, Bishop of Jerusalem" is the title for the article, I would suggest:
```
                <title xml:lang="gez" xml:id="t1">ድርሳን፡ በእንተ፡ ክብሮሙ፡ ለ፳፬፡ ካህናተ፡ ሰማይ፡</title>
                <title xml:lang="gez" corresp="#t1" type="normalized">Dǝrsān baʾǝnta kǝbromu la-24 kāhǝnāta samāy</title>
                <title xml:lang="en" corresp="#t1">Homily on the Glory of the 24 Heavenly Priests</title>
<title xml:lang="en" xml:id="t2" type="main">Homily on the glory of the 24 Heavenly Priests by Cyril, Bishop of Jerusalem</title>
```

File: new/LIT5895DersanaLedata.xml [hunk 5]
```diff
@@ -0,0 +1,92 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5895DersanaLedata" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+              <title xml:lang="gez" xml:id="t1">ድርሳነ፡ ልደታ፡ ለማርያም፡</title>
+              <title xml:lang="gez" corresp="#t1" type="normalized">Dǝrsana lǝdetā la-Maryam</title>
```
Comment (thea-m) [inline new/LIT5895DersanaLedata.xml:9]: Dǝrsānā lǝdatā la-Māryām

File: new/LIT5898DersanaMikael.xml [hunk 6]
```diff
@@ -0,0 +1,76 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5898DersanaMikael" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+              <title xml:lang="en" corresp="#t1">Homily on the Archangel Michael</title>
+              <title xml:lang="gez" xml:id="t2">ድርሳን፡ በእንተ፡ መልአክ፡ ክቡር፡ ሚካኤል</title>
+              <title xml:lang="gez" corresp="#t2" type="normalized">Dǝrsān baʾǝnta kǝbur malʾak Mikāʾel</title>
```
Comment (thea-m) [inline new/LIT5898DersanaMikael.xml:10]: malʾak kǝbur

File: new/LIT5898DersanaMikael.xml [hunk 7]
```diff
@@ -0,0 +1,76 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5898DersanaMikael" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+              <title xml:lang="en" corresp="#t1">Homily on the Archangel Michael</title>
```
Comment (thea-m) [inline new/LIT5898DersanaMikael.xml:8]: same as above. I think we should stick to the guidelines title structuring (Geez is t1, etc.). Then the title we will want to have printed can be added with type="main"

File: new/LIT5900DersanaMikaelTheo.xml [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5900DersanaMikaelTheo" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+              <title xml:lang="en" corresp="#t1">Homily on the Archangel Michael by Theodosius of Alexandria for 12 Ḫedār</title>
+              <title xml:lang="gez" xml:id="t2">ድርሳነ፡ በእንተ፡ ዕበዩ፡ ወክብሩ፡ ለሚካኤል</title>
+              <title xml:lang="gez" corresp="#t2" type="normalized">Dǝrsān baʾǝnta ʿǝbayu wa-kǝbru la-Mikāʾel</title>
+              <title xml:lang="en" corresp="#t2">Homily on the Honourable Angel Michael</title>
```
Comment (thea-m) [inline new/LIT5900DersanaMikaelTheo.xml:11]: ...the greatness and glory of Michael?

File: 1001-2000/LIT1651Homily.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply 1001-2000/LIT1651Homily.xml:7]: Not really. Corrected.

File: 1001-2000/LIT1651Homily.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply 1001-2000/LIT1651Homily.xml:14]: Done

File: 3001-4000/LIT3058RepCh338.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply 3001-4000/LIT3058RepCh338.xml:7]: I want to have it as: I worship Thee... (እሰግድ፡ ለኪ፡ እሰግድ፡ ለኪ፡ እሰግድ፡ ለኪ፡ ወእዌድሰኪ፡)
The words እሰግድ፡ ለኪ፡ are repeated three times. "I worship Thee..." is a kind of common English title.

File: 4001-5000/LIT4958Priests.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply 4001-5000/LIT4958Priests.xml:7]: Yes. But I would prefer t1 English followed by t2 Ethiopic (so, title 1 is not the exact translation of title 2). With homilies, it is always better to provide an author, it he is mentioned. Will it come out like this?

File: new/LIT5895DersanaLedata.xml (hunk 5 above)
Comment (DenisNosnitsin1970) [reply new/LIT5895DersanaLedata.xml:9]: Dǝrsāna lǝdatā :)

File: new/LIT5900DersanaMikaelTheo.xml (hunk 8 above)
Comment (DenisNosnitsin1970) [reply new/LIT5900DersanaMikaelTheo.xml:11]: Yes.

File: 3001-4000/LIT3058RepCh338.xml (hunk 3 above)
Comment (thea-m) [reply 3001-4000/LIT3058RepCh338.xml]: then my proposal should work

File: 4001-5000/LIT4958Priests.xml (hunk 4 above)
Comment (thea-m) [reply 4001-5000/LIT4958Priests.xml]: Yes, without the "", because it is not a translation anymore

Comment (DenisNosnitsin1970) [reply 4001-5000/LIT4958Priests.xml]: Sorry, do not understand. No "" - where?

Comment (thea-m) [reply 4001-5000/LIT4958Priests.xml]: Sorry, in the PDF I mean

**Pull request merged**